web/src/pages/Dashboard.tsx

```diff
@@ -266,7 +266,7 @@ export const NotificationsMap = () => {
         left={0}
         top={0}
         overflow={isSmallScreen ? undefined : "auto"}
-        width={isSmallScreen ? "100%" : "30%"}
+        width={isSmallScreen ? "100%" : "31%"}
```

Comment:
Request from Jennifer as few users where facing weird word wrap due to small dashboard size